crud/registry_test.go

```diff
@@ -100,26 +101,26 @@ func TestRegistryCreate(t *testing.T) {
 	err := r.Register("foo", a)
 	assert.Nil(err)
 
-	res, err := r.Create("foo", "yolo")
+	res, err := r.Create(nil, "foo", "yolo")
```

Comment:
Why `nil`?

https://golang.org/pkg/context/ says
> Do not pass a nil Context, even if a function permits it. Pass context.TODO if you are unsure about which Context to use.

Reply:
These are unit tests, does it matter?
Maybe from a referential code perspective...

Reply:
ping @mflendrich

Reply:
The documentation for `context.Background` says

> It is typically used by the main function, initialization, and tests

Given that:
- the documentation says to never pass `nil`
- the documentation suggests that `context.Background` is intended for use in tests
- it's a matter of a single line `ctx := context.Background` and replacing `nil` by `ctx`
- functions that get a nil `context` will panic if they try to call any function on a nil context (and from the perspective of tests we shouldn't be assuming that they don't)

I can see strong evidence convincing that we should pass a valid context also in tests.

The cost is one `ctx := context.Background()` line per function and replacement of `nil`s with `ctx`s which doesn't look like a big deal to me.

Reply:
Okay. I've updated the code. The most convincing part was `nil` context point.